samples/Calculator/src/Calculator.Console/Calculator.Console.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <ProjectReference Include="..\Calculator.Core\Calculator.Core.csproj" />
```

Comment:
I would introduce $(RepoRoot) so if in future we move the project around we don't need to update all projects

Reply:
I would keep it simple without any props on main level. Also it is not helping a lot in case you change dependency project location. Event with RepoRoot you have to fix it.